CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ function(mbed_generate_map_file target)
          TARGET
              ${target}
          POST_BUILD
-             COMMAND ${Python3_EXECUTABLE} ${MBED_ROOT}/tools/memap.py -t ${MBED_TOOLCHAIN} -d 4 ${CMAKE_BINARY_DIR}/${target}${CMAKE_EXECUTABLE_SUFFIX}.map
+             COMMAND ${Python3_EXECUTABLE} ${MBED_ROOT}/tools/memap.py -t ${MBED_TOOLCHAIN} ${CMAKE_BINARY_DIR}/${target}${CMAKE_EXECUTABLE_SUFFIX}.map
```

Comment:
Why was this changed?

Reply:
I just changed to make sure map console prints the same depth and format as old mbed-CLI
```
| Module           |         .text |       .data |        .bss |
|------------------|---------------|-------------|-------------|
| [fill]           |       50(+50) |       0(+0) |     18(+18) |
| [lib]/c.a        |   4820(+4820) | 2108(+2108) |     89(+89) |
| [lib]/gcc.a      |     760(+760) |       0(+0) |       0(+0) |
| [lib]/misc       |     180(+180) |       0(+0) |     28(+28) |
| [lib]/nosys.a    |       16(+16) |       0(+0) |       0(+0) |
| main.cpp.obj     |       80(+80) |       0(+0) |     28(+28) |
| mbed-os/drivers  |     124(+124) |       0(+0) |       0(+0) |
| mbed-os/hal      |   1418(+1418) |       8(+8) |   130(+130) |
| mbed-os/platform |   4948(+4948) |       4(+4) |   239(+239) |
| mbed-os/targets  | 11280(+11280) |       8(+8) | 1028(+1028) |
| Subtotals        | 23676(+23676) | 2128(+2128) | 1560(+1560) |
Total Static RAM memory (data + bss): 3688(+3688) bytes
Total Flash memory (text + data): 25804(+25804) bytes


```

Reply:
I think this warrants a separate commit.

Reply:
it should be at least in separate commit

Reply:
Done, please re-review